llpc/lower/llpcSpirvLowerRayQuery.cpp

```diff
@@ -1612,7 +1612,7 @@ void SpirvLowerRayQuery::createIntersectBvh(Function *func) {
 // Create sample gpu time
 //
 void SpirvLowerRayQuery::createSampleGpuTime(llvm::Function *func) {
-  assert(func->getBasicBlockList().size() == 1);
+  assert(std::distance(func->begin(), func->end()) == 1);
```

Comment:
Would `func->size()` work here instead?

Reply:
Good point! I did not know that existed.